stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -316,6 +316,14 @@ enum TypeCode {
   ///   WitnessV1Type {
   ///   }
   kWitnessV1Type = 26,
+
+  ///   FloatF8E4M3FNUZV1Type {
+  ///   }
+  kFloatF8E4M3FNUZV1Type = 27,
+
+  ///   FloatF8E5M2FNUZV1Type {
+  ///   }
+  kFloatF8E5M2FNUZV1Type = 28,
```

Comment:
How about we keep this list ordered alphabetically, even if this would mean that the enum values are no longer sorted in ascending order?

Reply:
I got the opposite impression from the comment at the top of this enum
```C
/// This enum contains marker codes used to indicate which type is
/// currently being decoded, and how it should be decoded. The order of these
/// codes must not be changed, as any changes will break compatibility
/// with older bytecode.
```

Reply:
Oh, I see what you mean. Order the definitions, but preserve the values.

Reply:
This may make it difficult to figure out "what value should the next added be?" Personally in favor of leaving it in numeric order, sorting was nice as a starting point, but I think will add to confusion and risk gaps in the enum values longer term.

Reply:
Sounds good. Let's stick with sorting values in ascending order.